File: front/src/admin/types/excursion.ts
export interface Excursion {
  id: number;

  title: string;
  city: string;

  description: string | null;

  price: number;

  duration: string;

  language: string;

  category: string;

  mainPhotoPath: string | null;

  rating: number;
  reviewCount: number;
}

File: api/Models/Excursion.cs
namespace DyplomBooking2026.Models
{
    public class Excursion
    {
        public int Id { get; set; }
        public string Title { get; set; } = "";
        public string City { get; set; } = "";
        public string ImagePath { get; set; } = "";
        public decimal Price { get; set; }
        public string Category { get; set; } = "";
        public string Duration { get; set; } = "";
        public double Rating { get; set; }
        public string Description { get; set; } = "";
    }
}
